app/controllers/items_controller.rb

```diff
@@ -2,4 +2,13 @@ class ItemsController < ApplicationController
   def index
     @items = Item.all
   end
+
+  def show
+    @item = Item.find(params[:id])
+    if @item.active?
+      @partial = 'partials/add_to_duffel'
+    else
+      @partial = 'partials/sold_out'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.